usr/initiator.c

```diff
@@ -453,7 +448,7 @@ static void iscsi_flush_context_pool(struct iscsi_session *session)
 static void
 __session_destroy(iscsi_session_t *session)
 {
-	log_debug(1, "destroying session");
+	log_debug(1, "destroying session %d", session->id);
```

Comment:
"session %d" is inconsistent with everything added to this point as "session:%d"